pkg/cmd/image/push.go

```diff
@@ -149,10 +150,21 @@ func Push(ctx context.Context, client *containerd.Client, rawRef string, options
 		return err
 	}
 
-	if err = signutil.Sign(rawRef, options.GOptions.Experimental, options.SignOptions); err != nil {
+	img, err := client.ImageService().Get(ctx, ref)
+	if err != nil {
+		return err
+	}
+	refSpec, err := reference.Parse(ref)
+	if err != nil {
+		return err
+	}
+	signRef := fmt.Sprintf("%s@%s", refSpec.String(), img.Target.Digest.String())
+	logrus.Infof("signing %s", signRef)
```

Comment:
nit: maybe move the log into `signutil.Sign`, and only log if `Provider != none`. Otherwise when sign is not enabled (i.e., `none`) we will only have a single log `signing xxx@xxx` which indicates we're signing something but no further info (in fact the signing is skipped).

Reply:
upps! I just added this for debugging purposes, let's remove this, thank you.